libusb/os/darwin_usb.c

```diff
@@ -170,6 +170,55 @@ static enum libusb_error darwin_to_libusb (IOReturn result) {
   }
 }
 
+uint32_t get_running_version(void) {
+  char os_version_string[64];
+  size_t os_version_string_len = sizeof(os_version_string) - 1;
```

Comment:
These variable declarations best inside the `#if` below I believe...

Reply:
Fixed.